### DISCUSSION TITLE
Gleam playground should load the Gleam home examples?!

### DISCUSSION THREAD

**Discussion by shahryarjb** [gleam-lang/gleam#1792] [Ideas & suggestions]
Hi, I hope you are doing well,
I saw the gleam home page that shows some example about:
```
Hello, world!
A basic web server
Multi-threaded hello world
```

After that I see [Get Started](https://gleam.run/getting-started) button, so I was happy to see the playground section until to test home page examples and get errors. I have no experience in gleam, and I was attracted by compiling to JS.
So at my first glance, I get errors. I think it is not good for new people, it is a bad experience.

```
error: Unknown module
  ┌─ ./src/main.gleam:5:8
  │
5 │ import gleam/otp/process
  │        ^^^^^^^^^^^^^^^^^ did you mean `list`?

No module has been found with the name `gleam/otp/process`.
```
I know it can be fixed, but for who? The person has Gleam Knowledge or me the person see this part first time.

Thank you for all your efforts 🌹

**Comment by lpil** [comment]
Hello! The playground is an unofficial project so we can't do much here.

If we had an official playground it would likely work the same way as the unofficial one and so it wouldn't be able to run the examples that use OTP as they only work when compiling to Erlang, and the playground compiles Gleam to JavaScript.

It would be cool to be able to compile to Erlang too but unless we get a sustantial amount more funding I don't think we'll be able to build and maintain the server infrastructure required for that any time soon.
